app/views/includes/head.njk

```diff
@@ -14,7 +14,13 @@
   ga('set', 'displayFeaturesTask', null);
   ga('set', 'transport', 'beacon');
 
-  ga('send', 'pageview');
+  var url = new URL(document.location.href);
+  var search_params = new URLSearchParams(url.search);
```

Comment:
Ahh hang on `URLSearchParams` [isn’t available](https://caniuse.com/#feat=urlsearchparams) in Internet Explorer 😩 

Just checked and 27% of traffic to selfservice is through IE so will have to write a fallback for it otherwise this fix won’t work on any queries in IE

Reply:
😩 do you have any suggestions?

Reply:
@jonheslop will `(new URL(document.location)).searchParams` work in IE?

Reply:
No, no support for that either https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams#Browser_compatibility 

Maybe we just go with a simpler regex

Reply:
Like this maybe? https://jsbin.com/kemacav/edit?js,console

Reply:
Just updated it to this - `/&email=[\w%.-]*/` which is a little more robust